src/Pages/ProductDetail/ProductDetail.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { Link, animateScroll as scroll} from 'react-scroll';
+import ImgPurchInfo from './Component/ImgPurchInfo';
+import ProductDescriptions from './Component/ProductDescriptions/ProductDescriptions';
+import Review from './Component/Review/Review'
+import Recommandations from './Component/Recommadations/Recommandations'
+import './ProductDetail.scss';
+
+const API = 'http://localhost:3000/data/data.json';
+
+class ProductDetail extends React.Component{
+
+    constructor(){
+        super();
+        this.state={
+            detailSelected: true,
+            reviewSelected: false,
+            qaSelected: false,
+            targetReached: false,
+            productData: {},
+            reviewList: []
+        };
+    }
+    
+    componentDidMount() {
+        // fetch(`http://10.168.1.149:8000/product/${this.props.match.params.id}`, {
+        fetch(API, {
+            method: "GET",
+        })
```

Comment:
get메소드는 default 값이므로 생략 가능합니다. 
```suggestion
        fetch(API)
```